sources/LocalizationEditor/Extensions/String+Extensions.swift

```diff
@@ -16,4 +16,19 @@ extension String {
     var capitalizedFirstLetter: String {
         return prefix(1).uppercased() + self.lowercased().dropFirst()
     }
+
+    var unescaped: String {
+        let entities = ["\0", "\t", "\n", "\r", "\"", "\'", "\\"]
+        var current = self
+        for entity in entities {
+            let descriptionCharacters = entity.debugDescription.dropFirst().dropLast()
```

Comment:
What is the purpose of this line? To my knowledge, debugDescription is not deterministic and could break your code whenever Apple chooses to change its output. What are you trying to do there?

Reply:
Also, In C, and therefore probably also in a .strings file, `\0` is not how a NUL char would be encoded. Instead, it would be `\000`. But then, no one probably puts NUL chars into .strings files, I hope :)

Reply:
The idea with the `debugDescription` was to get the the escaped representation, it kind of works (now), `print(entities.map({ $0.debugDescription.dropFirst().dropLast() }))` gets me

```
["\\0", "\\t", "\\n", "\\r", "\\\"", "\\\'", "\\\\"]
```

But yes, a better way is probably needed.